shared/icons/Cross.tsx

```diff
@@ -0,0 +1,20 @@
+export default function CrossIcon({ className, title }) {
+  return (
+    <svg
+      width="15"
+      height="15"
+      viewBox="0 0 15 15"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      className={className}
+    >
+      <title>{title}</title>
```

Comment:
Strikes me as odd that a SVG icon should have a title, what is this being used for?

Also, the idea behind this icons folder is that each icon can be colored using `text-<color>`, for that you would need to set stroke to currentColor i.e. `stroke="currentColor"`.

Reply:
the title attribute is being used for providing an explanation to the cross icon - further explanation - https://github.com/saving-satoshi/saving-satoshi/pull/349#issuecomment-1537974549